src/main/java/de/tum/in/www1/artemis/web/rest/hestia/CodeHintResource.java

```diff
@@ -62,11 +58,9 @@ public CodeHintResource(AuthorizationCheckService authCheckService, ProgrammingE
      */
     @GetMapping("programming-exercises/{exerciseId}/code-hints")
     @EnforceAtLeastEditor
+    @EnforceRoleInExercise(Role.EDITOR)
```

Comment:
Do you plan on keeping both annotations here?

Reply:
These used `EnforceRoleInExercise` instead of the (later added) `EnforceAtLeastEditorInExercise`.
I've updated them now.